shared/wallets/wallet-popup.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../common-adapters'
+import * as Styles from '../styles'
+import {compose, renameProp} from 'recompose'
+
+type WalletModalProps = {|
+  children: React.Node,
+  onClose: () => void,
+  containerStyle?: Styles.StylesCrossPlatform,
+  // Since the header is only displayed on mobile, its styles only apply to mobile.
+  headerStyle?: Styles.StylesCrossPlatform,
+  // Buttons to be placed in the bottom Button Bar.
+  // If none are included, the bar is not rendered.
+  bottomButtons?: Array<React.Node>,
```

Comment:
lets just make this a React.Node, you can pass in a fragment so we can avoid the array funkiness

Reply:
ah, k, lets leave it for now